## 📊 Descriptive Statistics

### Measures of Central Tendency
- **Mean:** Average of values
- **Median:** Middle value
- **Mode:** Most frequent value

In [2]:
import numpy as np
from statistics import mode

data = [5, 7, 3, 7, 9, 5, 7]

mean = np.mean(data)
median = np.median(data)
mode_val = mode(data)

print("Mean:", mean)
print("Median:", median)
print("Mode:", mode_val)

Mean: 6.142857142857143
Median: 7.0
Mode: 7


 # Mean (Average) Explanation

The **Mean** (often called the average) is a measure of central tendency.  
It is calculated using the formula:

$$
\text{Mean} = \frac{\text{Sum of all data points}}{\text{Number of data points}}
$$

---

## Example Dataset
Values: **5, 7, 3, 7, 9, 5, 7**
---

## Step 1: Sum of all values

$$
5 + 7 + 3 + 7 + 9 + 5 + 7 = 43
$$

---

## Step 2: Count of values
There are **7** numbers in the dataset.

---

## Step 3: Apply the formula

$$
\text{Mean} = \frac{43}{7} \approx 6.14
$$

---

## Final Answer
The **Mean** of the dataset is:

$$
\boxed{6.14}
$$

# [README](https://github.com/progressivepull/Statistics-Deep-Dives/blob/main/README.md)